src/v2/Apps/Collect/Routes/Collection/Components/CollectionArtworksFilter.tsx

```diff
@@ -20,10 +33,35 @@ interface CollectionArtworksFilterProps {
 
 export const CollectionArtworksFilter: React.FC<CollectionArtworksFilterProps> = props => {
   const { relay, collection, aggregations, counts } = props
-  const { slug } = collection
+  const { slug, query } = collection
+  const isArtistCollection = query?.artistIDs?.length === 1
```

Comment:
In this way, we check that this collection belongs to a certain artist

Reply:
This seems reasonable for the time being! I did a quick audit of our marketing collections with the query below to see if we had any collections with a single artist ID that _also_ had works by multiple artists and didn't find any examples of that. If we introduce that in the future, this could cause problems, but I'm not too concerned right now.

```graphql
{
  marketingCollections(size: 500) {
    title
    query {
      artistIDs
    }
  }
}
```